README.md

```diff
@@ -81,13 +81,30 @@ to have _you_ join us in iterating on it and helping us build it together!
 > (KIND)][kind] clusters. You can generate a new development cluster for
 > testing with `make build.cluster`.
 
-Deploy [Gateway API][gwapi] [CRDs][crds]:
+> **Note**: Currently our container images are under migration from a private repository.
+> At this moment, you should build and load images yourself.
+
+1. Deploy [Gateway API][gwapi] [CRDs][crds]:
 
 ```console
 kubectl apply -k https://github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.8.1
 ```
 
-Deploy:
+2. Build Blixt images:
+
+```console
+make build.all.images TAG=latest
+```
+
+3. Load images into your Kind cluster:
+
+```console
+make load.all.images TAG=latest
+```
+You may get an error like `Error from server (NotFound): namespaces "blixt-system" not found`, but please ignore it at this moment. The entire resources are created in the next step.
```

Comment:
At first I was a bit confused about this, but then taking a look at the `Makefile`, this is because of the rollout restart we added for the load which is convenient for quick building and testing. I had forgotten about that!

If you wanted to as a part of this PR, you could short circuit the error message by doing something like:

```diff
- kubectl -n blixt-system rollout restart deployment blixt-controlplane
+ kubectl -n blixt-system get deployment blixt-controlplane 2>/dev/null && \
+      kubectl -n blixt-system rollout restart deployment blixt-controlplane
```

I just wrote that up here in Github so I didn't test it, but I think something like that should work to only do the rollout if the thing exists, and quell the error output and ignore it if it doesn't.

Reply:
Good suggestion! I will modify the Makefile.

Reply:
Addressed in https://github.com/kubernetes-sigs/blixt/pull/177/commits/938c6866b583745402c5bf704161bfba9e141994